mypy/stubtest.py

```diff
@@ -1040,9 +1043,23 @@ def test_stubs(args: argparse.Namespace) -> int:
     options = Options()
     options.incremental = False
     options.custom_typeshed_dir = args.custom_typeshed_dir
+    options.config_file = args.config_file
+
+    plugins = []
+    if options.config_file:
+        def set_strict_flags() -> None: # not needed yet
```

Comment:
not sure if there's a better way to deal with this - seems like we just don't really need that param but I didn't want to pass in `lambda _: None`. open to suggestions here